preview-src/index.ts

```diff
@@ -101,6 +101,15 @@ function renderPullRequest(pr: PullRequest): void {
 	addEventListeners(pr);
 }
 
+function renderTimelineEvents(pr: PullRequest): void {
+	const timelineElement = document.getElementById(ElementIds.TimelineEvents)!;
```

Comment:
I think it would be easier to just do this once, at the top of the file. Instead of tracking `ElementIds`, we could just query for the elements and retain references to them.

If we go that route, we should probably make our lives easier by changing [`pullRequestOverview.ts`](src/github/pullRequestOverview.ts) to have the `<script>` after all the `<div>`s.

Reply:
Yeah, I agree. I think to make this webview maintainable we should ultimately break it down into classes for different UI components

Right now I don't think we can pull the getElementByIds up to the top of the file, since some of these elements are added as the PR loads. Let's handle this separately

Reply:
👍🏽